R/fit.R

```diff
@@ -730,9 +727,6 @@ CmdStanMCMC <- R6::R6Class(
 
     # override the CmdStanFit draws method
     draws = function(variables = NULL, inc_warmup = FALSE) {
-      if (!length(self$output_files(include_failed = FALSE))) {
```

Comment:
This error will also be thrown by `private$read_csv_` so it is unnecessary here. We also need to remove it or it will result in errors after creating the objects from read_cmdstan_csv.